python_modules/libraries/dagster-aws/dagster_aws/utils/__init__.py

```diff
@@ -40,6 +40,29 @@ class ResourceWithBoto3Configuration(ConfigurableResource):
     profile_name: Optional[str] = Field(
         default=None, description="Specifies a profile to connect that session"
     )
+    use_ssl: bool = Field(
+        default=True, description="Whether or not to use SSL. By default, SSL is used."
+    )
+    endpoint_url: Optional[str] = Field(
+        default=None, description="Specifies a custom endpoint for the Boto3 session."
+    )
+    verify: Optional[str] = Field(
+        default=None,
+        description=(
+            "Whether or not to verify SSL certificates. By default SSL certificates are verified."
```

Comment:
A description is slightly misleading vs how to use this. See other comment

Reply:
This is the description that I copied from the existing S3Resource
https://github.com/dagster-io/dagster/blob/master/python_modules/libraries/dagster-aws/dagster_aws/s3/resources.py#L36-L43